scripts/phpspy_build.sh

```diff
@@ -5,6 +5,16 @@
 #
 set -euo pipefail
 
+# TODO support aarch64
+if [ $(uname -m) != "x86_64" ]; then
+    mkdir -p /phpspy
+    touch /phpspy/phpspy
```

Comment:
Why? What's the point here? So the copy from the builder container doesn't fail?

Reply:
Yes, https://github.com/Granulate/gprofiler/pull/170#discussion_r695563141

Reply:
OK, so deleting this thread